extensions/lifecycle/objectProcessor/task.js

```diff
@@ -3,14 +3,15 @@
 const async = require('async');
 const werelogs = require('werelogs');
 const { errors } = require('arsenal');
-const {
-    DEFAULT_LIVE_ROUTE,
-    DEFAULT_READY_ROUTE,
-} = require('arsenal').network.probe.ProbeServer;
+// const {
```

Comment:
Are the probe-related changes wanted? Just checking but good for me if that's the case

Reply:
yes. I believe the plan is to move to the generic `ProbeServer` over the `HealthProbeServer`.
atm, `healthProbeServer` seems to also handle metrics which deviates from its original purpose.